lib/Dialect/TensorExt/Transforms/InsertRotate.cpp

```diff
@@ -0,0 +1,41 @@
+#include "include/Dialect/TensorExt/Transforms/InsertRotate.h"
+
+#include "include/Dialect/TensorExt/IR/TensorExtOps.h"
+#include "mlir/include/mlir/Dialect/Arith/IR/Arith.h"    // from @llvm-project
+#include "mlir/include/mlir/Dialect/Tensor/IR/Tensor.h"  // from @llvm-project
+#include "mlir/include/mlir/IR/Matchers.h"               // from @llvm-project
+#include "mlir/include/mlir/IR/PatternMatch.h"           // from @llvm-project
+#include "mlir/include/mlir/Transforms/GreedyPatternRewriteDriver.h"  // from @llvm-project
+
+namespace mlir {
+namespace heir {
+namespace tensor_ext {
+
+#define GEN_PASS_DEF_INSERTROTATE
+#include "include/Dialect/TensorExt/Transforms/Passes.h.inc"
+
+namespace alignment {
+// In an inner namespace to avoid conflicts with canonicaliation patterns
```

Comment:
nit: typo (curious why pre-submit didn't catch this!)

```suggestion
// In an inner namespace to avoid conflicts with canonicalization patterns
```

Reply:
Fixed